egs/wsj/s5/steps/nnet3/chain/get_egs.sh

```diff
@@ -65,6 +65,9 @@ online_ivector_dir=  # can be used if we are including speaker information as iV
 cmvn_opts=  # can be used for specifying CMVN options, if feature type is not lda (if lda,
             # it doesn't make sense to use different options than were used as input to the
             # LDA transform).  This is used to turn off CMVN in the online-nnet experiments.
+online_cmvn=false # set to 'true' to activate online_cmvn on the nnet3 input,
```

Comment:
Karel,  can you clarify in this comment that this uses (and I assume requires?) the config in conf/online_cmn.conf?

Reply:
and if it uses that config, I think the normal practice would be to make that configurable?  So we dont have difficulty to change those configuration values when running multiple experiments in the same dir?

Reply:
Hi, I don't refer the 'conf/online_cmn.conf' directly in this script. Instead, I am passing it in $cmvn_opts through train.py as --cmvn-opts="--config=conf/online_cmn.conf". So it is configurable from outside and it re-uses former mechanism for configuring the cmvn.